src/Emulator/Cores/linux-properties.csproj

```diff
@@ -3,6 +3,7 @@
 <PropertyGroup>
   <CompilerPath>/usr/bin/gcc</CompilerPath>
   <LinkerPath>/usr/bin/gcc</LinkerPath>
+  <ArPath>ar</ArPath>
```

Comment:
For consistency, can you make it `/usr/bin/ar`?

Reply:
Updated.  I didn't want to have a functional change, but if you want to keep it consistent I'll change it.

Reply:
I guess it does make sense, easier for future debugging